internal/config/transport/transport.go

```
@@ -63,6 +74,10 @@ type BootstrapDependencies struct {
 	Logger  log.Logger
 }
 
+func (c *Transport) ConfigureSSB() (transport.Transport, error) {
+	return nil, nil
```

Can we return an error here until SSB is implemented?

---

done